paf-mvp-demo-express/CONTRIBUTE.md

```diff
@@ -7,10 +7,10 @@ Start with [root contribute file](../CONTRIBUTE.md)
 1. create a new configuration file in `/src`
 2. add it in [index.ts](./src/index.ts)
 3. add the new configuration in [fake-hosts.ts](./scripts/fake-hosts.ts)
-4. generate an EC private key
-   1. For example: `openssl ecparam -name prime256v1 -genkey -noout -out private-key.pem`
-   2. copy its content to the config file, as `privateKey`
-5. generate the corresponding EC public key
-   1. For example: `openssl ec -in private-key.pem -pubout -out public-key.pem`
-   2. copy its content to the config file, as `publicKey`
-6. re-generate the SSL certificate (see [README.md](README.md))
+4. generate a pair of (public, private) keys with the `generateKeyPair()` function available [crypto/keys.ts](../paf-mvp-core-js/src/crypto/keys.ts)
```

Comment:
Running openssl commands seems the standard way of generating pub/private keys.
Will it be easy to use the js function from the cli to generate the keys?

Reply:
Nevermind, I didn't see https://github.com/criteo/onekey/pull/8/files